backend/app/adapter/gqlapi/args/input.go

```diff
@@ -0,0 +1,37 @@
+package args
```

Comment:
Considering changing name of this package, since it conflicts with resolver function's argument name.

Reply:
input